modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormInstanceSubmissionLimitStatusUtil.java

```diff
@@ -69,14 +69,10 @@ public static boolean isSubmissionLimitReached(
 		Optional<DDMFormInstanceRecordVersion> optional = stream.filter(
 			ddmFormInstanceRecordVersion ->
 				ddmFormInstanceRecordVersion.getStatus() !=
-					WorkflowConstants.STATUS_DRAFT
+					WorkflowConstants.ACTION_PUBLISH
```

Comment:
@Gabriel422-hue @SamZiemer I think the right one should be `WorkflowConstants.STATUS_PENDING` instead of `WorkflowConstants.ACTION_PUBLISH`. Even if they have the same values, it gets confusing with `ACTION_PUBLISH` because it's not a status.